src/trex/utils.py

```diff
@@ -13,3 +18,42 @@ def format(self, record):
         if record.levelno != logging.INFO:
             record.msg = "{}: {}".format(record.levelname, record.msg)
         return super().format(record)
+
+
+def molecule_list_2_df(mol_list: List[Molecule]) -> pd.DataFrame:
```

Comment:
Please don’t use the `2` abbrevation for `to`. It can be ambiguous. You could just name this function `molecules_to_dataframe`